css/contact.css

```diff
@@ -0,0 +1,225 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;600&display=swap');
+
+
+
+:root{
+    --primary:#0150FD;
+  --dark: #E6DACD;
+   --ligth:#F4ECE6;
+   --Black: #000
+   --White: #FFF
+  }
+*{
+  box-sizing: border-box;
+  margin: 0;
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+  color: black;
+}
+
+  
+  body{
+    background-color:whitesmoke;
+    font-family:'Poppins', sans-serif;
+  }
+  .grid-container{
+    display: grid;
+    grid-template-columns: repeat(1fr 1fr 1fr);
+    grid-template-rows: 80px auto 150px;
+  }
+
+
+.grid-header{
+    display: grid;
+    grid-gap: 20px;
+    grid-template-columns: 1fr 1fr;
+    grid-area:"header header nav-bar nav-bar ";
+    padding-bottom: 32px;
+    padding-top: 32px;
+    position: fixed;
+
+}
+.header{
+  padding-top: 32px;
+  padding-bottom: 32px;
+  display:grid;
+  grid-area:"header header" ;
+  grid-template-columns: 1fr 1fr;
+  text-align: left;
+  float: left;
+
+  
+}
+
+ h1{
+font-family: sans-serif;
+  display: flex;
+  float: left;
+  margin-left: 4vh;
+  font-size: 18px;
+  font-weight: 900;
+
+}
+span{
+  font-weight: 600;
+}
+
+.nav-bar ul{
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-area:"nav-bar  nav-bar nav-bar nav-bar" ;
+  grid-gap: 20px;
+  list-style: none;
+  text-decoration: none;
+  text-align: end;
+  justify-content: flex-end;
+  justify-content: space-between;
+  float: right;
+  margin-top: 10px;
+}
+.nav-bar a{g
+  display: block;
+  text-decoration: none;
+  text-align: center;
+}
+.nav-bar  ul li a:hover{
+  color:var(--primary)
+}
+
```

Comment:
Most of the above code could go in a CSS file shared between the pages, saves duplicating the code and avoids confusion futher down the line.